sentry-spring-boot-jakarta/src/test/kotlin/io/sentry/spring/boot/jakarta/SentryAutoConfigurationTest.kt

```diff
@@ -445,6 +447,15 @@ class SentryAutoConfigurationTest {
             }
     }
 
+    @Test
+    fun `when Spring MVC is not on the classpath, fallback TransactionNameProvider is configured`() {
```

Comment:
(l) would it make sense to also add a test to assert that, if MVC is used, the `SpringMvcTransactionNameProvider` is instantiated?

Reply:
Yeah we can add this kind of test later to not block this PR.